docs/kernel-policy.md

```diff
@@ -1,21 +1,35 @@
 # Firecracker's Kernel Support Policy
 
-Once officially supported kernel versions are supported for a **minimum of 2
-years**.
-
-We are validating the currently supported Firecracker releases as per
-[Firecracker’s release policy](../docs/RELEASE_POLICY.md). Starting with release
-`v1.0` each major and minor release will specify the supported kernel versions.
-Adding support for a new kernel version will result in a Firecracker release
-only if compatibility changes are required.
+Firecracker is tightly coupled with the guest and host kernels on which it is
```

Comment:
commit `chore(docs) Update kernel-policy.md`:
- revisit -> revisits
- dates on -> dates of

Reply:
fixed thanks

Reply:
Still: "This commits revisit" -> "This commit revisits".
Once at it: "host and guest kernel" -> "host and guest kernels".

Reply:
done hopefully for good, thanks.